projects/api/prisma/seed.ts

```diff
@@ -1,21 +1,49 @@
 import { PrismaClient } from '@prisma/client';
 
+import CreateSupplierDto from '@/suppliers/dto/create-supplier.dto';
+
 import { brands, categories, countries } from './data/products';
 
 const prisma = new PrismaClient();
 
 async function seed() {
```

Comment:
PLEASE, consider adding comments to explain complex logic or any non-obvious decisions in your code
This can make it easier for other developers (or even yourself) to understand the code in the future

Reply:
Initializing the database to insert initial data into the database (through Prisma maybe? or without it)